src/index.html

```diff
@@ -4,9 +4,321 @@
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/reset.scss">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a class="header__logo-link" href="#logo">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="moyo logo"
+        >
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active nav__link" href="#Apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Laptops & Computers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Gadgets">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Tablets">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Photo">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Video">Video</a>
+          </li>
+        </ul>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div class="card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE iMac 27 Retina 5K"
+          >
+          <a class="card__title" href="#">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+          <p class="card__desc-text">Product code: 195434</p>
+          <div class="star">
+            <ul class="stars stars--N">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p class="review">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <span class="card__text-price">Price:</span>
+            <span class="card__text-value">$2,199</span>
+          </div>
+          <button class="button">
```

Comment:
same - just make it element of `card` BEM block

Reply:
![Screenshot_4](https://github.com/mate-academy/layout_catalog/assets/149514552/23f7a0a5-e37d-4d7e-8085-ab18c479a383)